src/main/resources/application.yaml

```diff
@@ -8,31 +8,83 @@ management:
   endpoints:
     web:
       base-path: /
-      exposure:
-        include: health, info, prometheus
 
-#If you use a database then uncomment below lines and update db properties accordingly
 spring:
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
that smells, it means we dont control which bean is loaded

Reply:
discussed 👍